### PR TITLE
Convert Innate Spellcasting

### PR DESCRIPTION
**Pull request by bagelbits**
## Overview
This is based on our conversations in #4 but it only handles the Innate Spellcasting for simplicity.

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
Generally seems pretty good.  Doing this by hand or programmatically?

**Comment by bagelbits** [comment]
By hand. Especially since the Spell Name have some inconsistent casing. I could have automated it but it isn't worth it for only changing in 20 places. Also the spell name extraction would have been annoying.

File: 5e-SRD-Monsters.json [hunk 1]
```diff
@@ -8747,7 +8981,43 @@
...
+          ],
+          "spells": [
+            {
+              "name": "Detect Evil and Good",
+              "url": "http://www.dnd5eapi.co/api/spells/78",
+              "usage": {
+                "type": "at will"
+              }
+            },
+            "1/Day Each: Commune, Raise Dead",
```
Comment (ogregoire) [inline 5e-SRD-Monsters.json]: Same here, this line should probably be deleted.

File: 5e-SRD-Monsters.json [hunk 2]
```diff
@@ -8606,7 +8797,50 @@
...
+          "spells": [
+            {
+              "name": "Nondetection",
+              "notes": "Self only",
+              "url": "http://www.dnd5eapi.co/api/spells/213",
+              "usage": {
+                "type": "at will"
+              }
+            },
+            "1/day each: blindness/deafness, blur, disguise self",
```
[the remark this answers is not in the document]
Comment (bagelbits) [reply 5e-SRD-Monsters.json]: Heck! Thank you.

**Comment by bagelbits** [comment]
@ogregoire Anything else?

**Comment by ogregoire** [comment]
@bagelbits No, those two lines were the only issues I saw. Excellent job overall!

**Pull request merged**